app/controllers/posts_controller.rb

```diff
@@ -20,12 +30,11 @@ def create
         post_is_valid = post.valid? #checking if the post fields are filled in 
         if post_is_valid  
             post.save # saves the parameters inside the database
-           return redirect_to '/home' 
+           return redirect_to '/posts' 
```

Comment:
cool so here you can use the Rails url helpers instead of hardcoding the URL in a string